### PR TITLE
[Accuracy diff No. 110, 111] Fix accuracy diff for paddle.nn.functional.binary_cross_entropy, paddle.nn.functional.binary_cross_entropy_with_logits API

### PR DESCRIPTION
**Pull request by NKNaN**
重新修改了一下
https://github.com/PFCCLab/PaddleAPITest/pull/267

### CODE REVIEW

**Comment by cangtianhuang** [comment]
@NKNaN 请解决一下冲突吧～

**Review by wanghuancoder — APPROVED** [review]
